src/controllers/update/update.director.correspondence.address.link.controller.ts

```diff
@@ -0,0 +1,13 @@
+import { NextFunction, Request, Response } from "express";
+import { UPDATE_DIRECTOR_CORRESPONDENCE_ADDRESS_PATH, UPDATE_DIRECTOR_RESIDENTIAL_ADDRESS_PATH } from "../../types/page.urls";
+import { Templates } from "../../types/template.paths";
+import { getCorrespondenceLink, postCorrespondenceLink } from "../shared.controllers/director.correspondence.address.link.controller";
+
+
+export const get = async (req: Request, res: Response, next: NextFunction) => {
+  getCorrespondenceLink(req, res, next, Templates.UPDATE_DIRECTOR_CORRESPONDENCE_ADDRESS_LINK, UPDATE_DIRECTOR_CORRESPONDENCE_ADDRESS_PATH)
+};
+
+export const post = async (req: Request, res: Response, next: NextFunction) => {
+  postCorrespondenceLink(req, res, next, Templates.UPDATE_DIRECTOR_CORRESPONDENCE_ADDRESS_LINK, {yes: UPDATE_DIRECTOR_RESIDENTIAL_ADDRESS_PATH, no: UPDATE_DIRECTOR_RESIDENTIAL_ADDRESS_PATH}, UPDATE_DIRECTOR_CORRESPONDENCE_ADDRESS_PATH, true)
```

Comment:
Possibly making this nextPageUrl optional since yes and no redirect to the same path?